source/macos/AdaptiveCards/AdaptiveCards/Views/ACRChoiceButton.swift

```diff
@@ -279,3 +292,30 @@ extension DispatchQueue {
         asyncAfter(deadline: .now() + duration, execute: execute)
     }
 }
+class ACRHyperLinkTextView: NSTextView {
+    private var clickOnLink = false
+    override var intrinsicContentSize: NSSize {
+        guard let layoutManager = layoutManager, let textContainer = textContainer else {
+            return super.intrinsicContentSize
+        }
+        layoutManager.ensureLayout(for: textContainer)
+        let size = layoutManager.usedRect(for: textContainer).size
+        let width = size.width + 2
+        return NSSize(width: width, height: size.height)
+    }
+    override func draw(_ dirtyRect: NSRect) {
+        super.draw(dirtyRect)
+    }
```

Comment:
Does this need override if you just call super in it?

Reply:
nope, not required.